acceptance/tests/fixtures/bases/static-metrics-app/deployment.yaml

```diff
@@ -14,6 +14,8 @@ spec:
       annotations:
         'consul.hashicorp.com/connect-inject': 'true'
         'consul.hashicorp.com/connect-service': 'server'
+        'consul.hashicorp.com/service-metrics-path': '/metrics'
+        'consul.hashicorp.com/service-metrics-port': '9090'
```

Comment:
Why we didn't need to do this before?

Reply:
We had some code in consul_sidecar/metrics_config that set the default path to `/metrics` and the default port was the one that was used to register the service.  From the port code:

```
// The annotationPort is the port used to register the service with Consul.
// If that has been set, it'll be used as the port for getting service
// metrics as well, unless overridden by the service-metrics-port annotation.
```

I created a jira ticket for the missing defaults.